### PR TITLE
Trigger block-changed event on update text alignment 

### LINKED ISSUE #57: Text alignment does not trigger block-changed event
Reproduction: change the alignment.

Why it matters: without triggering change event, an auto saving functionality cannot detect and update change of the editor.

### LINKED ISSUE #68: Build doesn't work
Hi, 
```
Couldn't find any versions for "@editorjs/helpers" that matches "workspace:*"
? Please choose a version of "@editorjs/helpers" from this list: (Use arrow keys)
❯ 0.0.1 
```

quote refers to editorjs/dom and it refers to editorjs/helpers which can't be resolved.

---

Looks like this "workspace:*" is valid for pnpm/yarn2, but nextjs doesn't support yarn2 and also yarn is more popular than pnpm.

### PR DESCRIPTION
**Pull request by aavishkarmishra**
Resolve #57 
Resolve #68 

Update :  Find current quote block using quote element and dispatch change when alignment is updated

@neSpecc  Please review

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -410,5 +416,10 @@ export default class Quote implements BlockTool {
    */
   _toggleTune(tune: Alignment) {
     this._data.alignment = tune;
+
+    // Dispatch change if quoteElement already exists
+    if (this._quoteElement) {
+      this.api.blocks.getBlockByElement(this._quoteElement)?.dispatchChange();
```
Comment (neSpecc) [inline src/index.ts]: you don't need to find a Block via API. It is passed to the tools contructor

https://editorjs.io/tools-api/#class-constructor

Comment (aavishkarmishra) [reply src/index.ts]: Done 👍🏼

**Review by neSpecc — DISMISSED** [review]
Increment a patch version, please

**Comment by aavishkarmishra** [comment]
Includes changes from #70  also

@e11sy @neSpecc

**Comment by e11sy** [comment]
resolve conflicts

**Comment by aavishkarmishra** [comment]
> resolve conflicts

Resolved Conflicts @e11sy @neSpecc

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@editorjs/quote",
-  "version": "2.7.1",
+  "version": "2.8.0",
```
Comment (neSpecc) [inline package.json]:
```suggestion
  "version": "2.7.2",
```

**Review by neSpecc — APPROVED** [review]

**Pull request merged**